h2o-py/h2o/frame.py

```diff
@@ -306,6 +306,27 @@ def filter_na_cols(self, frac=0.2):
         """
         return ExprNode("filterNACols", self, frac)._eager_scalar()
 
+    def filter_type(self,type="numeric"):
+        """ Filter frame by column type.
+
+        Parameters
+        ----------
+        type : str
+            A character string indicating which column type to filter by. This must be one of the following:
+                "numeric"      - Numeric, but not categorical or time
+                "categorical"  - Integer, with a categorical/factor String mapping
+                "string"       - String column
+                "time"         - Long msec since the Unix Epoch - with a variety of display/parse options
+                "uuid"         - UUID
+                "bad"          - No none-NA rows (triple negative! all NAs or zero rows)
+        Returns
+        -------
+          A list of column indices that correspond to "type"
+        """
+        assert_is_type(type, str)
+        assert_is_type(self,H2OFrame)
+        return ExprNode("filterByType", self, type)._eager_scalar()
```

Comment:
Wait, why do you return a list of column indices instead of a new frame? I thought that would be the most common use case?